be/src/olap/wal/wal_table.cpp

```diff
@@ -263,19 +259,13 @@ Status WalTable::_handle_stream_load(int64_t wal_id, const std::string& wal,
     return st;
 }
 
-Status WalTable::_replay_one_txn_with_stremaload(int64_t wal_id, const std::string& wal,
+Status WalTable::_replay_one_wal_with_streamload(int64_t wal_id, const std::string& wal,
                                                  const std::string& label) {
-    bool success = false;
 #ifndef BE_TEST
-    auto st = _handle_stream_load(wal_id, wal, label);
-    auto msg = st.msg();
-    success = st.ok() || st.is<ErrorCode::PUBLISH_TIMEOUT>() ||
-              msg.find("LabelAlreadyUsedException") != msg.npos;
+    return _handle_stream_load(wal_id, wal, label);
 #else
-    success = k_stream_load_exec_status.ok();
```

Comment:
k_stream_load_exec_status should be reserved, wal_manager_test.cpp need this to simulate repaly wal fail case